src/main/java/htsjdk/samtools/SAMSequenceRecord.java

```diff
@@ -148,6 +148,23 @@ public SAMSequenceRecord setMd5(final String value) {
         return this;
     }
 
+    /**
+     * Set the M5 attribute for this sequence using the provided sequenceBases.
+     * @param sequenceBases sequence bases to use to compute the MD5 for this sequence
+     * @return the update sequence record
+     */
+    public SAMSequenceRecord setComputedMd5(final byte[] sequenceBases) {
+        try {
+            final MessageDigest md5Digester = MessageDigest.getInstance("MD5");
+            if (md5Digester != null) {
```

Comment:
This method seems broken.  It doesn't actually digest the bases. Also, the bases should be normalized.

Why not use SequenceUtil.calculateMD5String()?